### PR TITLE
Fix Red Hat 8 selinux issue

### LINKED ISSUE #622: Server start failures in RHEL-8 as RHEL-8 is not yet in support matrix
In RHEL-8 when omi-1.4.2-2 is installed, omiserver is unable to start, and it fails to function properly. From syslogs, "SELinux (a kernel security module) is preventing systemd from read access on the file omiserver.pid." omiserver crashes are intermittent.

### PR DESCRIPTION
**Pull request by JumpingYang001**
Fixes #622 , @microsoft/omi-devs , I have fixed the issue, could you help to review this PR? thanks.

New installation output:
```
[root@ost64-rh8-02 ~]# rpm -i omi-1.6.0-230.ssl_110.ulinux.x64.rpm
Creating omiusers group ...
Creating omi group ...
Creating omi service account ...

************************************************************
* Warning: The certificate and keyfile were not generated  *
* since they already exist.                                *
************************************************************
2019-05-14 03:23:25 : Crontab not configured to update omi keytab automatically. Skip unconfigure
2019-05-14 03:23:25 : Crontab configured to update omi keytab automatically
Checking if cron is installed...
Checking if cron/crond service is started...
Set up a cron job to OMI logrotate every 15 minutes
System appears to have SELinux installed, attempting to install selinux policy module for logrotate
  Trying /usr/share/selinux/packages/omi-logrotate/omi-logrotate.pp ...
  Labeling omi log files ...
Configuring OMI service ...
Created symlink /etc/systemd/system/multi-user.target.wants/omid.service → /usr/lib/systemd/system/omid.service.

```

### CODE REVIEW

**Comment by Divya273** [comment]
Can we get any beta build with this fix to validate from our side?

**Comment by JumpingYang001** [comment]
@Divya273 only private build now, will send you later.
And we found the current fix has a bug on redhat5, so investigating redhat5 issue too, other platforms works fine.

**Comment by JumpingYang001** [comment]
Fixed redhat5 don't have [open selinux permission setting](https://danwalsh.livejournal.com/49762.html) issue, and also keep other platforms logic not change.

**Comment by Divya273** [comment]
When can we expect the build with fix?

**Comment by JumpingYang001** [comment]
@Divya273 I am not sure, you can ask Saroj and Manjari, they make decisions.

**Comment by JumpingYang001** [comment]
@sarojcare can you help to review and approve the PR? we have verified on all Linux platforms, thanks.

**Comment by JumpingYang001** [comment]
@sarojcare , we have tested on all x64 Linux platform for this fixes.
for `SEPOLICY_DIR_EL6`, we never install `omi-logrotate.el6.te` even we built it, check this install only `omi-logrotate.pp` : https://github.com/microsoft/omi/blob/master/Unix/installbuilder/datafiles/Linux.data#L350

**Comment by JumpingYang001** [comment]
we remove `SEPOLICY_DIR_EL6` and add `SEPOLICY_DIR_EL5`, the reason is that:
1. `omi-logrotate.el6.te` never used.

2. redhat 5 don't have open permission setting. allow init_t var_t:file open;
other higher OS have open permission setting, so we add red hat 5 selinux different setting.

**Comment by JumpingYang001** [comment]
@sarojcare ,  Ping ...

**Comment by JumpingYang001** [comment]
@sarojcare I have updated the PR as we discussed, can you help to review it when you are free? thanks.

**Comment by JumpingYang001** [comment]
@sarojcare , Ping ...

File: Unix/installbuilder/datafiles/Linux.data [hunk 1]
```diff
@@ -328,6 +330,22 @@ if ${{PERFORMING_UPGRADE_NOT}}; then
 fi
 
 %Postinstall_1500
+
+# Special handling for Red Hat 5 don't have open selinux permission
+# return 0, means is redhat 5; 1 means others.
+is_redhat5(){
```
Comment (bbsaurav) [inline Unix/installbuilder/datafiles/Linux.data]: Do we need this we no longer support RedHat5

Comment (JumpingYang001) [reply Unix/installbuilder/datafiles/Linux.data]: our universal build for rpm and deb is redhat 5.

Comment (JumpingYang001) [reply Unix/installbuilder/datafiles/Linux.data]: I don't know if we can drop it or not.
@sarojcare how do you think?

Comment (JumpingYang001) [reply Unix/installbuilder/datafiles/Linux.data]: I see 2016 UR 7 support RedHat 5.

Comment (bbsaurav) [reply Unix/installbuilder/datafiles/Linux.data]: we should think of upgrading our universal build for rpm and deb from RedHat 5

Comment (JumpingYang001) [reply Unix/installbuilder/datafiles/Linux.data]: due to some history reason, we still use RedHat 5 for build server.
an old machine list:

Supported Platforms | glibc | Machine
-- | -- | --
Centos5x64(Not support) | glibc-2.5-123.el5_11.3 | osd64-ct5-01
centos6x64 | glibc-2.12-1.209.el6_9.2.x86_64 | osd64-ct61-01
oracle6x64 | glibc-2.12-1.7.el6.x86_64 | It seems we don’t have pbuild for it.
Rhel5x64 | glibc-2.5-12 | osd64-rh5-01
Rhel6x64 | glibc-2.12-1.7.el6.x86_64 | osd64-rh6-01

**Comment by sarojcare** [comment]
Added deepak to review it.

**Pull request merged**